internal/gcloud/cloudrunservice.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/util"
+	"strings"
+	"unicode"
+)
+
+// Maximum length of a Cloud Run Service name.
+const maxCloudRunServiceNameLen = 53
+
+// Length of the random suffix at the end of the Cloud Run Service name.
+const cloudRunServiceNameRandSuffixLen = 10
```

Comment:
nit: you can combine these into a singe block
```suggestion
const {
   // Maximum length of a Cloud Run Service name.
    maxCloudRunServiceNameLen = 53
    // Length of the random suffix at the end of the Cloud Run Service name.
    cloudRunServiceNameRandSuffixLen = 10
}
```

Additionally, your comment and variable names are pretty much the same things - you can either use shorter var names or remove the comments (Go tends to favor short but meaningful variable names - `maxNameLen` is probably proficient enough given the file is "cloud run service").

Reply:
+1 to all this.